src/app.html

```diff
@@ -1,15 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8" />
-		<!-- <link rel="icon" href="%sveltekit.assets%/favicon.png" /> -->
-		<link rel="preconnect" href="https://fonts.googleapis.com">
-		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-		<link href="https://fonts.googleapis.com/css2?family=Playfair:wght@300&family=Wix+Madefor+Display&display=swap" rel="stylesheet">
-		<meta name="viewport" content="width=device-width" />
-		%sveltekit.head%
-	</head>
-	<body data-sveltekit-preload-data="hover">
-		<div style="display: contents">%sveltekit.body%</div>
-	</body>
-</html>
+
+<head>
+	<meta charset="utf-8" />``
+	<!-- <link rel="icon" href="%sveltekit.assets%/favicon.png" /> -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link
+		href="https://fonts.googleapis.com/css2?family=Fira+Mono:wght@500&family=Fira+Sans:ital,wght@0,400;1,700&display=swap"
+		rel="stylesheet">
+	<meta name="viewport" content="width=device-width" />
+	%sveltekit.head%
+</head>`
+
+<body data-sveltekit-preload-data="hover">
+	<div style="display: contents">%sveltekit.body%</div>
+</body>
+
+</html>
```

Comment:
probably should \n while we are here.

Reply:
weird that the default html formatter in vscode doesn't do it, but fixed

Reply:
yeah, there are other files missing them as well